elyra/pipeline/processor_local.py

```diff
@@ -205,8 +205,13 @@ def process(self, operation: Operation):
                 filepath,
                 **additional_kwargs
             )
+        except papermill.PapermillExecutionError as pmee:
+            self.log.error(f'Internal error executing {file_name}: {str(pmee.ename)}' +
+                           f'{str(pmee.evalue)} in [{str(pmee.exec_count)}]')
+            raise RuntimeError(f'{str(pmee.ename)} {str(pmee.evalue)} in [{str(pmee.exec_count)}]') from pmee
         except Exception as ex:
-            raise RuntimeError(f'Internal error executing {filepath}: {ex}') from ex
+            self.log.error(f'Internal error executing {file_name}: {str(ex)}')
+            raise RuntimeError('Internal error executing notebook') from ex
```

Comment:
We should include the notebook name in the error.
```suggestion
            raise RuntimeError(f'Internal error executing {file_name}') from ex
```

Reply:
Ah sorry, I must have misunderstood this piece in our earlier conversation. I removed the file name in the popup message because the exception message one up the chain already includes the node name, which already is the file name (sans extension). 

```
Error processing operation load_data: Internal error executing load_data.ipynb: ...
```

Do you think we should keep both in?

Reply:
I'm not sure it will always be the case that the node name will always be derived from the file name and I feel including the actual file name is an extra level of detail that could be useful.  We might also find other "internal execution errors" that are based on other things - not necessarily related to the node/file name. 

That said, everyone has opinions and mine aren't necessarily correct. :smile:  I'm going to as for @ptitzler to review as well.

Reply:
I agree. Unlike in earlier releases the node name and the file name can be entirely different.

Reply:
Alright, that's helpful! I was wondering if there were cases where node name != file name. I'll add the file name to each of these errors, and remove the word "internal" since it's unnecessary.